flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -259,4 +265,37 @@ public static StructLikeSet actualRowSet(Table table, Long snapshotId, String...
 
     return dataFiles;
   }
+
+  public static Map<Long, List<DataFile>> snapshotToDataFiles(
+      Table table)
+      throws IOException {
+    table.refresh();
+    Map<Long, List<DataFile>> result = Maps.newHashMap();
+    List<ManifestFile> manifestFiles = table.currentSnapshot().dataManifests();
+    for (ManifestFile manifestFile : manifestFiles) {
+      try (ManifestReader<DataFile> reader = ManifestFiles.read(manifestFile, table.io())) {
+        List<DataFile> dataFiles = IteratorUtils.toList(reader.iterator());
```

Comment:
I see a warning in IDE here, using the `Lists.newArrayList(reader)` could fix the warning.

Reply:
Good catch, done